src/helloComponent.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {StyleSheet, Text, View} from 'react-native';
+
+const HelloComponent = () => {
+  return (
+    <View style={styles.HelloContainer}>
+      <Text style={styles.helloText}> Hello </Text>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  HelloContainer: {
```

Comment:
You should name your style using camel case ("helloContainer"). 

- Pascal case ("HelloComponent") should be used by : Classes, Types, Interfaces, React Components
- Camel case should be used by : function names, variable names, keys in objects (like a style name or a jss property (ex: paddingHorizontal))